camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeper.java

```diff
@@ -587,7 +618,18 @@ public int run(String[] args) throws Exception
     props.putAll(cmd.getOptionProperties("D"));
 
     init();
+
+    boolean forcedExit = Boolean.parseBoolean(props.getProperty("camus.forced.exit.enabled"));
+
+    if (forcedExit) {
+        long forcedExitTimeout = Long.parseLong(props.getProperty("camus.forced.exit.timeout"));
+        int forcedExitStatus = 0;
```

Comment:
Otherwise we'll have to and check the logs each time to see if this related to these strange hang ups or something else.. Everything seems to finish correctly before the hang up happens so perhaps it's a `success`?